samigo/samigo-app/src/webapp/js/calcQuestInputValidator.js

```diff
@@ -15,8 +15,8 @@ $( document ).ready(function() {
     });
   });
 
-  $('.calculatedQuestionInput').popover({
-    trigger: 'focus'
+  $('.calculatedQuestionInput').on('shown.bs.popover', function () {
+    $(this).delay(3000).popover('hide');
```

Comment:
This works? Isn't this the old jquery BS-3 way?

Reply:
I thought bs-5 wasn't using jquery anymore?

Can't we just use bootstrap.Popover.getInstance(this).hide(); ?

and specify the delay in the popover constructor options?